crypto/s2n_drbg.h

```diff
@@ -40,20 +40,8 @@ struct s2n_drbg {
     /* The current DRBG 'value' */
     uint8_t v[S2N_DRBG_BLOCK_SIZE];
 
-    /* Function pointer to the entropy generating function. If it's NULL, then
-     * s2n_get_urandom_data() will be used. This function pointer is intended
-     * ONLY for the s2n_drbg_test case to use, so that known entropy data can
-     * fed to the DRBG test vectors.
-     */
-    S2N_RESULT (*entropy_generator) (struct s2n_blob *);
-
-    /* Also used only by the unit tests: which generation of the DRBG is this.
-     * This number is incremented every time we reseed.
-     */
-    uint32_t generation;
-
-    /* Indicate if the DRBG instance offers prediction resistance and should reseed after every call */
-    uint8_t use_prediction_resistance;
+    /* Used only by the unit tests: how many times has entropy been mixed in */
+    uint32_t mixes;
```

Comment:
`uint64_t`?

Reply:
I changed it to `uint64_t`. I'm not really sure how concerned we should be about overflow on `mixes`. I don't have a great sense of how often the new entropy gets mixed in during normal usage.